Sources/MeiliSearch/TenantTokenClaims/ApiKeyClaim.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import JWTKit
+
+internal struct ApiKeyClaim: JWTClaim, Equatable {
```

Comment:
`ApiKeyClaim` makes me think there is an `apiKey` claim in the payload. Could we call it `apiKeyPrefixClaim` ?

Reply:
Good catch, I'll change it to `ApiKeyPrefixClaim` :)